manifests/service.pp

```diff
@@ -17,6 +18,12 @@
     }
 
     foreman::dynflow::worker { 'worker': }
+
+    if $jobs_worker_hosts_queue {
```

Comment:
I think this needs a way to disable the worker if it's false, otherwise it'll never be stopped.